.angular-cli.json

```diff
@@ -63,6 +63,9 @@
         "component": {
             "inlineStyle": false,
             "inlineTemplate": false
+        },
+        "build": {
+            "showCircularDependencies": false
```

Comment:
Is this a good thing to leave as `false`?

Reply:
Besides this one line I think the rest looks good